examples/v1beta1/pipelineruns/pipelinerun-with-when-expressions.yaml

```diff
@@ -66,6 +66,16 @@ spec:
           - name: echo
             image: ubuntu
             script: 'echo file exists'
+    - name: echo-file-missing
+      when:
+        - input: "$(tasks.check-file.results.status)"
```

Comment:
i wonder if we should call this something other than "status" to reduce confusion around being able to execute if a previous task fails

Reply:
made the update on the naming in this open pr: https://github.com/tektoncd/pipeline/pull/3194

should i move the naming change here?